Update de.json

razer4908 opened the pull request:
some logical grammar change in german, should be good now :)

oznu commented:
Hi @razer4908,

Thanks heaps for this. I actually renamed and added a load more translations strings and this merge won't work 😭 

I've cherry picked the items you suggested I changed and added them to the current `de.json` file, if you would like to review it now and update this PR that would be most appreciated. 

I think I have pretty much all of the UI covered now, apart from some rarely seen errors. I have been using this app to help edit the de and en json files at the same time:

https://www.codeandweb.com/babeledit

razer4908 commented:
I corrected myself a bit. I updated config Ui a few minutes ago and saw some mistakes, this should be fine now. I also edited your additional lines if needed. Do you think the Issue, opened by BoardLord82  is possible? I do not like to give our neighbours full controll of my house. They just need the garden pump :D 

Have a nice Weekend !

razer4908 commented:
Well, with UTF-8 chars like ä,ö or ü are now showing correctly. German Final Language Pack is attached. 

[de.zip](https://github.com/oznu/homebridge-config-ui-x/files/2064160/de.zip)
